metpy/calc/tests/test_tools.py

```diff
@@ -28,14 +29,46 @@ def test_nearest_intersection_idx():
     assert_array_equal(truth, nearest_intersection_idx(y1, y2))
 
 
-def test_find_intersections():
+intersection_directions = ['all', 'increasing', 'decreasing', 'decdreasing']
+
+
+@pytest.mark.parametrize('direction', intersection_directions)
```

Comment:
Yeah, I'm not worried about its complexity complaints. Just fix up the `parametrize`.